shared/react-native/ios/Keybase/PushPrompt.m

```diff
@@ -0,0 +1,39 @@
+//
+//  PushPrompt.m
+//  Keybase
+//
+//  Created by Daniel Ayoub on 6/5/18.
+//  Copyright © 2018 Keybase. All rights reserved.
+//
+
+#import "PushPrompt.h"
+#import <Foundation/Foundation.h>
+#import <React/RCTBridgeModule.h>
+#import <UserNotifications/UserNotifications.h>
+
+@implementation PushPrompt
+
+#pragma mark - RCTBridgeModule
+
+RCT_EXPORT_MODULE(PushPrompt);
+
++ (BOOL)requiresMainQueueSetup
+{
+  return NO;
+}
+
```

Comment:
this is fine but just a general FYI, if we do native stuff you always have to make sure if it needs the main thread or not and if so to add the right methodQueue. In this case its safe to call this from any thread but many calls are main thread only still